base_vat_optional_vies/models/res_partner.py

```diff
@@ -39,7 +42,20 @@ def vies_vat_check(self, country_code, vat_number):
 
     @api.constrains("vat", "country_id")
     def check_vat(self):
+        self.update({"vies_passed": False})
         for partner in self:
             partner = partner.with_context(vat_partner=partner)
             super(ResPartner, partner).check_vat()
         return True
+
+    @api.model
+    def _build_vat_error_message(self, country_code, wrong_vat, record_label):
+        return "\n" + _(
+            "The VAT number [%(wrong_vat)s] for %(record_label)s does not seem to be valid. "
+            "\nNote: the expected format is %(expected_format)s",
+            wrong_vat=wrong_vat,
+            record_label=record_label,
+            expected_format=_ref_vat.get(
+                country_code, "'CC##' (CC=Country Code, ##=VAT Number)"
```

Comment:
Shouldn't this be translated too?

```suggestion
                country_code, _("'CC##' (CC=Country Code, ##=VAT Number)")
```

Reply:
This is how Odoo is building the msg: https://github.com/odoo/odoo/blob/15.0/addons/base_vat/models/res_partner.py#L224

I'm just showing that Odoo shows. So the customers are going to see exactly the same message as before (without VIES check). Also, it's a fallback and doesn't appear so much.

If you consider that I must translate this, I apply the suggestion directly.